openedx/core/djangoapps/user_api/helpers.py

```diff
@@ -129,8 +129,10 @@ class FormDescription(object):
     ALLOWED_RESTRICTIONS = {
         "text": ["min_length", "max_length"],
         "password": ["min_length", "max_length", "min_upper", "min_lower",
-                     "min_punctuation", "min_symbol", "min_numeric", "min_alphabetic"],
+                     "min_punctuation", "min_symbol", "min_numeric", "min_alphabetic",
+                     "readonly"],
```

Comment:
This change can be removed -- we won't ever be making the password read-only with SSO (since there's no password at all).

Reply:
Done